core/toolbox.js

```diff
@@ -648,9 +679,20 @@ Blockly.Toolbox.prototype.refreshSelection = function() {
   }
 };
 
+/**
+ * Select the first toolbox category if no category is selected.
+ * @package
```

Comment:
Just curious why this only selects the first category if no category is selected? Also could we change the name to selectFirstCategory?

Reply:
That's the desired behaviour from Blockly.navigation. If we're going from WS -> Toolbox, select first category. If from Flyout -> Toolbox, no need to select a category as there would have had to be a category selected for the flyout to be open.

Reply:
As for name change, it's not necessarily a "category" that we're selecting although that's how we're using it. But since this is on BaseNode, I'll try to keep it generic like the rest of the methods here.

Reply:
But this is on the Toolbox not BaseNode right? So the name could be more specific?

Reply:
Yes, you're right! Will fix.